labs/dxp/7.x/installation-and-upgrades/02-setting-up-liferay-dxp/installing-and-managing-apps/blacklisting-apps.md

```diff
@@ -0,0 +1,79 @@
+# Blacklisting Apps
+
+The *bundle blacklist* is a convenient way to manage multiple apps, OSGi bundles (modules), and plugin WAR installations. The list is an [OSGi configuration](https://help.liferay.com/hc/en-us/articles/360029131591-System-Settings#exporting-and-importing-configurations) that DXP uses to uninstall apps. You can set the list in the Control Panel or by using a `.config` file. It saves you the trouble of managing app, module, and plugin installations individually with the [Application Manager](./managing-apps.md) or [Gogo shell](https://help.liferay.com/hc/en-us/articles/360029070351-Using-the-Felix-Gogo-Shell).
+
+The blacklist can be exported from the Control Panel, so that you have the correct file name and syntax. Modifying the file and deploy it to DXP has these additional effects:
```

Comment:
@Alec-Shay - I wrote these two sentences poorly and the part about exporting a file to get the correct file name is an incomplete thought. I'd prefer replacing both sentences with something like the text we use in Blacklisting Components...

| The blacklist can be exported from the Control Panel to an OSGi configuration (`.config`) file. Modifying the file and deploying it to DXP has these additional effects: